### PR TITLE
add Kafka federator docs

### PR DESCRIPTION
**Pull request by djfritz**

### CODE REVIEW

File: ingesters/ingesters.md [hunk 1]
```diff
@@ -410,6 +410,46 @@ Common configuration errors for the Federator include:
 * Enforcing certification validation when upstream indexers or Federators do not have certificates signed by a trusted certificate authority (see the `Insecure-Skip-TLS-Verify` option)
 * Mismatched Ingest-Secret for downstream ingesters
 
+## Kafka Federator
+
+Gravwell also provides a Kafka Federator, that behaves exactly like the Federator, except that it uses Kafka as an upstream transport. Ingesters can connect to it just with Federator, and ingested entries will be put into Kafka Topics as messages.
```
Comment (traetox) [inline ingesters/ingesters.md:415]: `Ingesters can connect to it just with Federator, and ingested entries` this right?

File: ingesters/ingesters.md [hunk 2]
```diff
@@ -410,6 +410,46 @@ Common configuration errors for the Federator include:
 * Enforcing certification validation when upstream indexers or Federators do not have certificates signed by a trusted certificate authority (see the `Insecure-Skip-TLS-Verify` option)
 * Mismatched Ingest-Secret for downstream ingesters
 
+## Kafka Federator
```
Comment (traetox) [inline ingesters/ingesters.md:413]: can we break this out into its own file for future maintainability?

File: ingesters/ingesters.md [hunk 3]
```diff
@@ -410,6 +410,46 @@ Common configuration errors for the Federator include:
...
+
+[IngestListener "enclaveA"]
+	Ingest-Secret = CustomSecrets
+	Cleartext-Bind = 0.0.0.0:4423
+	Tags=windows
+	Tags=syslog
+	Topic=testing
+	Kafka-Header-Set=headers
+
+[KafkaHeaders "headers"]
```
Comment (traetox) [inline ingesters/ingesters.md:446]: would you add a quick `NOTE:` section calling out these special `$` header values that resolve to items in an entry please.

**Review by traetox — CHANGES_REQUESTED** [review]
minor nitpicks

**Comment by djfritz** [comment]
@traetox address all nits, PTAL

**Review by traetox — APPROVED** [review]
LGTM

**Pull request merged**